src/index.html

```diff
@@ -1,14 +1,45 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin ="#">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+          src="images/logo.png"
+          alt="#" class="logo__img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item"><a href="#" class="nav_link is-active">Apple</a></li>
+          <li class="nav_item"><a href="#" class="nav_link">Samsung</a></li>
+          <li class="nav_item"><a href="#" class="nav_link">Smartphones</a></li>
+          <li class="nav_item">
+            <a href="#"
+              class="nav_link"
+              data-qa="hover">Laptops & Computers
+            </a>
+          </li>
+          <li class="nav_item"><a href="#" class="nav_link">Gadgets</a></li>
+          <li class="nav_item"><a href="#" class="nav_link">Tablets</a></li>
+          <li class="nav_item"><a href="#" class="nav_link">Photo</a></li>
+          <li class="nav_item"><a href="#" class="nav_link">Video</a></li>
+        </ul>
+      </nav>
+    </header>
   </body>
+
 </html>
```

Comment:
```suggestion
  </body>
</html>
```
Don't leave empty lines between parent and child elements